### PR TITLE
Updated codebase based on linter

### PR DESCRIPTION
**Pull request by abassey2**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

Update the codebase based on ESLint.

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/234
- References https://github.com/retaildevcrews/wcnp/issues/215

### CODE REVIEW

File: client/src/components/App/index.js [hunk 1]
```diff
@@ -110,37 +110,40 @@ const loadTests = [
...
+        <LoadClients handleOpen={handleOpen} />
+        {isOpen && (
+          <LoadClientDetails
+            handleClose={handleClose}
+            currClientDetails={currClientDetails}
+          />
+        )}
       </LoadClientContext.Provider>
       <ConfigsContext.Provider value={{ configs, loadTests }}>
-        <Configs></Configs>
```
Comment (joheec) [inline client/src/components/App/index.js:143]: I love this change

File: client/src/components/Configs/index.js [hunk 2]
```diff
@@ -1,66 +1,93 @@
...
-    const loadTestSelect = (id) => {
-        selectedLoadTestId === id ? setselectedLoadTestId(-1) : setselectedLoadTestId(id);
-    }
+  const configSelect = (id) => () => {
+    return currConfigId === id ? setcurrConfigId(-1) : setcurrConfigId(id);
+  };
 
-    const config = configs.find((c) => c.id === currConfigId);
-    const loadTestPath = loadTests.find((c) => c.id === selectedLoadTestId);
+  const loadTestSelect = (id) => {
```
Comment (joheec) [inline client/src/components/Configs/index.js]: Automatic return

File: client/src/components/Configs/index.js [hunk 3]
```diff
@@ -1,66 +1,93 @@
...
-}
+  return (
+    <div className="main">
+      <div id="configpath">
+        {config && (
+          <p>
+            <b>{config.name} &nbsp;</b>
+          </p>
+        )}
+        {config && loadTestPath && config.id === loadTestPath.configId && (
```
Comment (joheec) [inline client/src/components/Configs/index.js]: Do we need the config?

File: client/src/components/Configs/index.js [hunk 4]
```diff
@@ -1,66 +1,93 @@
...
+          <p>
+            <b>
+              {"> "}
+              {loadTestPath.name}{" "}
```
Comment (joheec) [inline client/src/components/Configs/index.js]: Can you use a string literal here?

File: client/src/components/Configs/index.js [hunk 5]
```diff
@@ -1,66 +1,93 @@
...
-        currConfigId === id ? setcurrConfigId(-1) : setcurrConfigId(id);
-    }
+  const [currConfigId, setcurrConfigId] = useState(-1);
+  const [selectedLoadTestId, setselectedLoadTestId] = useState(-1);
 
-    const loadTestSelect = (id) => {
-        selectedLoadTestId === id ? setselectedLoadTestId(-1) : setselectedLoadTestId(id);
-    }
+  const configSelect = (id) => () => {
+    return currConfigId === id ? setcurrConfigId(-1) : setcurrConfigId(id);
```
Comment (joheec) [inline client/src/components/Configs/index.js]: Add loadTestSelect to configSelect to handle reset of currConfigId to include automatic reset of currLoadTestId

**Review by joheec — CHANGES_REQUESTED** [review]

File: client/src/components/LoadClients/index.js [hunk 6]
```diff
@@ -1,14 +1,13 @@
...
-import { LoadClientContext } from '../../contexts';
+import { useContext, useState } from "react";
+import { LoadClientContext } from "../../contexts";
 
+import "./styles.css";
 
-import "./styles.css"
-
-const LoadClients = props => {
+const LoadClients = (handleOpen) => {
```
Comment (joheec) [inline client/src/components/LoadClients/index.js]: not props?

**Review by joheec — CHANGES_REQUESTED** [review]

File: client/src/components/Configs/index.js (hunk 4 above)
Comment (abassey2) [reply client/src/components/Configs/index.js]: Is this better?

File: client/src/components/Configs/index.js [hunk 7]
```diff
@@ -10,18 +10,22 @@ const Configs = () => {
...
-    return selectedLoadTestId === id
-      ? setselectedLoadTestId(-1)
-      : setselectedLoadTestId(id);
-  };
+  const loadTestSelect = (id) => (selectedLoadTestId === id ? setselectedLoadTestId(-1) : setselectedLoadTestId(id));
 
   const config = configs.find((c) => c.id === currConfigId);
   const loadTestPath = loadTests.find((c) => c.id === selectedLoadTestId);
 
+  console.log(loadTestPath);
```
Comment (joheec) [inline client/src/components/Configs/index.js]: remove

File: client/src/components/Configs/index.js [hunk 8]
```diff
@@ -10,18 +10,22 @@ const Configs = () => {
   const [selectedLoadTestId, setselectedLoadTestId] = useState(-1);
 
   const configSelect = (id) => () => {
-    return currConfigId === id ? setcurrConfigId(-1) : setcurrConfigId(id);
+    if (currConfigId === id) {
```
Comment (joheec) [inline client/src/components/Configs/index.js]: Use ternary

File: client/src/components/Configs/index.js (hunk 4 above)
Comment (joheec) [reply client/src/components/Configs/index.js]: {\`> ${loadTestPath.name} \`}

**Review by joheec — CHANGES_REQUESTED** [review]

**Pull request merged**